ui/client/components/Process/State/ProcessState.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import {ProcessStateType, ProcessType} from "../ProcessTypes"
+import StateIcon from "./StateIcon"
+import {CSSTransition, SwitchTransition} from "react-transition-group"
+import ProcessStateUtils from "./ProcessStateUtils"
+import {unknownDescription, unknownName} from "../ProcessMessages"
+
+type State = {
+  animationTimeout: {
+    enter: number;
```

Comment:
Please remove unecessarry semicolons

Reply:
Semicolons are reqiured at type definition ;) I know - it's strange..

Reply:
Theye were, but I've changed it in lint, now there are not required

Reply:
Hmmm, okey. So I will change it ;)